### ISSUE TITLE
VSCode freezing/hanging when trying to indent improperly formatted rule in .less file

### ISSUE THREAD

**Issue by prufrock123** [Microsoft/vscode#19739]
- VSCode Version: 1.9
- OS Version: 10.10.5

Steps to Reproduce:

1. in a .less file, If I paste the following:
color: 'green', fontSize: '22px
into a rule, VSCode completely locks up.
2. I Don't even have time to fix the CSS into the correct format. I was just copying it over from a JSX file.
3. It's weird. 
4. I'm running the Sublime Text Keymap and beautiful extension. But I don't see how that would be related. Neither should be actively doing anything.

**Comment by aeschli** [comment]
I'm not able to reproduce with 1.9.
![19739](https://cloud.githubusercontent.com/assets/6461412/22607736/71e55338-ea59-11e6-875e-4cab2b09f498.gif)

Is this always reproducible for you? 
- Open the developer tools (Help > Toggle Developer Tools') and check the console.
- Open the output view and check for output in the 'Extensions' channel.
- Run without any extension (run code with `code --disable-extensions`) and see it that makes a difference.

**Comment by prufrock123** [comment]
@aeschli Thank-you for looking into this. I just updated to 1.9.1 and I can no longer reproduce. I also updated all of my extensions. Possibly something related to that was the issue.  Or maybe just a bizarre confluence of issues resulted in the problem.

Either way, I can close this issue out as resolved. 

Thanks again for your efforts. Keep up the great work. VSC is by far my favorite editor now. You're all rockstars in my book.
